lib/components/ToasterDialog/ToasterDialog.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import { Meta, StoryFn } from '@storybook/react'
+
+import { ToastContainer } from 'react-toastify'
+
+import { default as ToasterDialogComponent } from './ToasterDialog'
+import { DialogProvider } from '../../context'
+import React from 'react'
+import Utils from '../../utils'
+import Button from '../Button'
+import { CloseButton } from '../index'
+
+import 'react-toastify/dist/ReactToastify.css'
+
+interface StoryArgs {
+  successMessage: string
+  errorMessage: string
+}
+
+interface toasterStoryArgs {
+  successToastMessage: string
+  errorToastMessage: string
+}
+
+const DialogStory = (args: StoryArgs) => (
+  <>
```

Comment:
do we need the Fragment if we have the common div parent?

Reply:
Nope, we don't need, there was context before but moved to the story itself. Removed :)